very_many_backends/multi_backend.py

```diff
@@ -262,6 +262,20 @@ class MultipleBackends(stress.StressTest):
 
     base_port = 16384
 
+    def setUp(self):
+        super().setUp()
+        self.create_servers()
+        self.configure_tempesta()
+        # Cleanup part
+        self.addCleanup(self.cleanup_interfaces)
+        self.addCleanup(self.cleanup_servers)
+        self.addCleanup(self.cleanup_tempesta)
```

Comment:
```suggestion
        self.addCleanup(self.cleanup_interfaces)
```
`cleanup_servers` and `cleanup_tempesta` are called from `StressTest`.